### DISCUSSION TITLE
TrueNAS iGPU (Alder Lake)

### DISCUSSION THREAD

**Discussion by roklipni** [immich-app/immich#14448] [Q&A]
Has anyone managed to get iGPU working on TrueNAS Scale (24.10) with the immich app? I used the passthrough GPU option, but facial recognition still uses CPU. The docs - https://immich.app/docs/features/hardware-transcoding/ - don't have anything specific for the 12th Gen Intel GPU. Should I have enabled something?

What I can confirm is that it does work with Plex - using _Passthrough available (non-NVIDIA) GPUs_ so the OS can utilise it.

Can anyone help?

**Comment by psavan143** [comment]
I am also on TrueNAS Scale (24.10) and facing same issue. It seems system is not detecting the GPU. Maybe it is not yet supported by TrueNAS yet. Can you check if you have /dev/dri directory? Run 'cd /dev/dri' command. Also, what is your system? I have N150 which has igpu for sure.

**Reply by roklipni** [reply]
I am using Aoostar WTR Pro with N100 and I do have the /dev/dri directory. It surely has an iGPU as Plex can use it to transcode (confirmed).

**Comment by jonathan-bdlf** [comment]
Any updates 2 years later ?

**Comment by roklipni** [comment]
Not that I know, sorry.
